pkg/reconciler/managed/reconciler.go

```diff
@@ -83,6 +87,31 @@ const (
 	reasonReconciliationPaused event.Reason = "ReconciliationPaused"
 )
 
+var supportedManagementPolicies = []sets.Set[xpv1.ManagementAction]{
+	// default
+	sets.New[xpv1.ManagementAction](xpv1.ManagementActionAll),
+	//// all
```

Comment:
```suggestion
	// all
```

Should these have just two `/`?

Also, I wonder if these comments should be a little more detailed? Or dropped? Reading through them right now I'm not sure they're providing much extra context over the code.

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Updated the comments with some more context